### PR TITLE
Build Break: Fix missed lint

### PR DESCRIPTION
**Pull request by anthony-murphy**
New lint rule came in between the last build of a previous pr, so a lint failure slipped in. this fixes it.

### CODE REVIEW

**Comment by anthony-murphy** [comment]
feel free to just squash after approval to unblock PR builds

**Review by heliocliu — APPROVED** [review]

**Pull request merged**
